bundles/de.uka.ipd.sdq.simulation.abstractsimengine.desmoj/src/de/uka/ipd/sdq/simulation/abstractsimengine/desmoj/DesmoJExperiment.java

```diff
@@ -21,8 +22,14 @@ public DesmoJExperiment(final DesmoJModel model) {
         super(model);
 
         this.desmojModel = model;
-        this.experiment = new Experiment(model.getConfiguration().getNameExperimentRun(), TimeUnit.NANOSECONDS,
-                TimeUnit.SECONDS, new SingleUnitTimeFormatter(TimeUnit.SECONDS, TimeUnit.NANOSECONDS, 9, false));
+        this.experiment = new Experiment(model.getConfiguration().getNameExperimentRun(), ".", //<- Path for storing output that defaults to .
+        		TimeUnit.NANOSECONDS,
+                TimeUnit.SECONDS, 
+                new SingleUnitTimeFormatter(TimeUnit.SECONDS, TimeUnit.NANOSECONDS, 9, false), 
+                new ArrayList<String>(), //<- Empty to avoid nasty file creation
+                new ArrayList<String>(), //<- Empty to avoid nasty file creation
+                new ArrayList<String>(), //<- Empty to avoid nasty file creation
+                new ArrayList<String>());//<- Empty to avoid nasty file creation

```

Comment:
I'd vote for pulling these into something like a `private static final ArrayList<String> EMPTY_ARRAY_LIST`.